src/app/organizations/state/organization-members.service.ts

```diff
@@ -93,9 +93,11 @@ export class OrganizationMembersService {
           // appears as if he still has permissions to edit.
           if (this.userQuery.getValue().user) {
             const currentUserId = this.userQuery.getValue().user.id;
-            const canEdit = organizationUsers.some(user => user.id.userId === currentUserId && user.accepted);
+            const canEdit = organizationUsers.some(
+              user => user.id.userId === currentUserId && user.accepted && user.role !== OrganizationUser.RoleEnum.MEMBER
```

Comment:
Optional, because this works, but I feel it's generally better to check for an explicit role, rather than infer its capabilities from what the role is not.

If we add a fourth role some day and miss updating this code, then the user will be able to edit. 

If we change this code now, add a fourth role some day and forget to update this code, the user will not be able to edit. Better to have too few permissions than too many.

Also, I know this would be caught by the API in any case, so there wouldn't really be a security hole, but still.